framework/test/unit/modules/fee/module.spec.ts

```diff
@@ -187,6 +220,9 @@ describe('FeeModule', () => {
 		});
 
 		it('should burn the used fee when addressFeePool is not defined', async () => {
+			when(tokenMethod.userAccountExists)
+				.calledWith(expect.anything(), context.header.generatorAddress, feeModule['_tokenID'])
+				.mockResolvedValue(true as never);
```

Comment:
This is the option 2 from my comment above.

Reply:
I improve the test description to clearly identify it.